### PR TITLE
GCP TF provider

### PR DESCRIPTION
**Pull request by trezorg**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -72,14 +78,13 @@ jobs:
           name: Run Go Azure tests
           command: |
             export PREFIX="$(cat /dev/urandom | tr -dc 'a-z0-9' | fold -w 5 | head -n 1 | sed -E 's/^([0-9]+)(.*)/\2\1/g')"
-            VERSION=$(git describe --tags --abbrev=8 --exact-match 2>/dev/null || echo "${POLCADOT_PROVIDER_VERSION}")
-            [[ "${VERSION}" =~ "^azure" ]] || VERSION="${POLCADOT_PROVIDER_VERSION}"
-            export VERSION=${VERSION#*-}
-            make test-azure
-          environment:
-            - POLCADOT_PROVIDER_VERSION: 0.1.0
+            VERSION=$(git describe --tags --abbrev=8 --exact-match 2>/dev/null || echo "${AZURE_POLCADOT_PROVIDER_VERSION}")
```
Comment (ArseniiPetrovich) [inline .circleci/config.yml:81]: It's polKadot. Please, make sure you have fixed it everywhere in the code

File: .circleci/config.yml [hunk 2]
```diff
@@ -111,14 +116,34 @@ jobs:
...
+      - checkout
+      - run:
+          name: "Publish GCP polkadot provider release on GitHub"
+          command: |
+            GO111MODULE=off go get github.com/tcnksm/ghr
+            VERSION=$(git describe --tags --abbrev=8 --exact-match 2>/dev/null || echo "${GCP_POLCADOT_PROVIDER_VERSION}")
+            [[ "${VERSION}" =~ "^gcp" ]] || VERSION="${GCP_POLCADOT_PROVIDER_VERSION}"
+            VERSION=${VERSION#*-} make build-gcp-provider
+            mv -v ./terraform-provider-polkadot ./terraform-provider-polkadot-gcp
+            ghr -t ${GITHUB_TOKEN} -u ${CIRCLE_PROJECT_USERNAME} -r ${CIRCLE_PROJECT_REPONAME} -c ${CIRCLE_SHA1} -delete ${VERSION} ./terraform-provider-polkadot-gcp
```
Comment (ArseniiPetrovich) [inline .circleci/config.yml:142]: Providers seems to be a standalone units. Can we store them in the separate repo?

**Review by ArseniiPetrovich — CHANGES_REQUESTED** [review]

File: .circleci/config.yml (hunk 1 above)
Comment (trezorg) [reply .circleci/config.yml:81]: Fixed

File: .circleci/config.yml (hunk 2 above)
Comment (trezorg) [reply .circleci/config.yml:142]: As discussed, removed into a separate issue

**Pull request merged**